Corzbank.Services/LoggerManager.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Corzbank.Services.Interfaces;
+using NLog;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Corzbank.Services
+{
+    public class LoggerManager : ILoggerManager
+    {
+        private static ILogger logger = LogManager.GetCurrentClassLogger();
+        public LoggerManager()
+        {
+        }
+        public void LogDebug(string message)
+        {
+            logger.Debug(message);
+        }
+        public void LogError(string message)
+        {
+            logger.Error(message);
+        }
+        public void LogInfo(string message)
+        {
+            logger.Info(message);
+        }
+        public void LogWarn(string message)
+        {
+            logger.Warn(message);
+        }
```

Comment:
Add empty lines between methods for consistency

Reply:
resolved